Test/ElasticScale.ShardManagement.UnitTests/ScenarioTests.cs

```diff
@@ -1237,6 +1237,119 @@ public void BasicScenarioRangeShardMaps()
             Assert.IsTrue(success);
         }
 
+        [TestMethod()]
+        [TestCategory("ExcludeFromGatedCheckin")]
+        public void ListShardMapPerformanceCounterValidation()
+        {
+            if (PerfCounterInstance.HasCreatePerformanceCategoryPermissions())
```

Comment:
Calling Assert.Inconclusive() if test does not have permissions to create perf counter category.